charts/gardener-extension-kupid/templates/service.yaml

```diff
@@ -3,8 +3,9 @@ kind: Service
 metadata:
   name: {{ include "name" . }}
   namespace: {{ .Release.Namespace }}
-{{-  if .Values.ignoreResources }}
   annotations:
+    networking.resources.gardener.cloud/from-world-to-ports: '[{"protocol":"TCP","port":{{ .Values.webhookConfig.serverPort }}}]'
```

Comment:
@rfranzke Don't we now need another policy for scraping metrics, since #52 was merged and metrics port is well-defined now?
Something like `networking.resources.gardener.cloud/from-policy-allowed-ports: '[{"port":{{ .Values.metricsPort }},"protocol":"TCP"}]'` and/or `networking.resources.gardener.cloud/from-policy-pod-label-selector: all-seed-scrape-targets`, right?

Reply:
Yes, right, thanks. Done with the latest commit.